### ISSUE TITLE
Script via ZIP import doesn't work

### ISSUE THREAD

**Issue by Taroslord** [RivinHD/ScriptToButton#4]
Scripts are imported wrong. See the example below. Looks like you place UTF code.
The original code is this:
```
import bpy

selection = bpy.context.selected_objects #Get selected objects

for o in selection: #Loop through selected objects
    bpy.context.view_layer.objects.active = o #Set object in loop as active 
    if bpy.context.view_layer.objects.active.type == 'MESH': #if object type is MESH then...
        bpy.ops.mesh.customdata_custom_splitnormals_clear()
```

The version you import looks like this:
`b"import bpy\r\n\r\nselection = bpy.context.selected_objects #Get selected objects\r\n\r\nfor o in selection: #Loop through selected objects\r\n    bpy.context.view_layer.objects.active = o #Set object in loop as active \r\n    if bpy.context.view_layer.objects.active.type == 'MESH': #if object type is MESH then...\r\n        bpy.ops.mesh.customdata_custom_splitnormals_clear()\r\n"`

The ZIP file include the right code.

Info: Importing Python files works fine.

**Comment by RivinHD** [comment]
Solved in Version 2.1.3

**Comment by Taroslord** [comment]
Looks good so far!
Thank you!
